components/collator/Cargo.toml

```diff
@@ -44,7 +44,7 @@ databake = { version = "0.1.3", path = "../../utils/databake", optional = true,
 serde = { version = "1.0", default-features = false, features = ["derive", "alloc"], optional = true }
 
 icu_collator_data = { version = "~1.3.0", path = "data", optional = true }
-icu_locid_transform = { version = "1.2.0", path = "../../components/locid_transform", features = ["data"], optional = true }
+icu_locid_transform = { version = "1.2.0", path = "../../components/locid_transform", features = ["compiled_data"], optional = true }
```

Comment:
Nit (here and elsewhere): This is a bit misleading; I would prefer if you included `icu_locid_transform` without `compiled_data` up here and then included it with the feature down in the `compiled_data` section:

```
icu_locid_transform = { version = "1.2.0", path = "../../components/locid_transform", optional = true }

...

compiled_data = ["dep:icu_collator_data", "icu_normalizer/compiled_data", "dep:icu_locid_transform", "icu_locid_transform/compiled_data"]
```

Reply:
+1

Reply:
This is preexisting and I'd like to keep this PR a clean rename.